src/util/sss_python.c

```diff
@@ -39,7 +39,11 @@ sss_exception_with_doc(char *name, char *doc, PyObject *base, PyObject *dict)
     }
 
     if (doc != NULL) {
+#ifdef IS_PY3K
+        docobj = PyUnicode_FromString(doc);
+#else
         docobj = PyString_FromString(doc);
+#endif
```

Comment:
Change is correct per se. But we should never hit this in python3. Because this code is in #165 se branch of  following code:
```
`    23 PyObject *
     24 sss_exception_with_doc(char *name, char *doc, PyObject *base, PyObject *dict)
     25 {
     26 #ifdef HAVE_PYERR_NEWEXCEPTIONWITHDOC
     27     return PyErr_NewExceptionWithDoc(name, doc, base, dict);
     28 #else
```

Could we try to find out why the macro HAVE_PYERR_NEWEXCEPTIONWITHDOC was not properly set at configure time? we should really use `PyErr_NewExceptionWithDoc` on python2.7+

Reply:
@lslebodn, this is a valid issue but not related to this patch.
Please, would you mind opening a ticket for it? If you don't have the time, let me know and I will open the ticket and assign it to myself.

Reply:
Thanks, I was already wondering why this wasn't an issue before.

Maybe the whole issue is just a fluke because there was a glibc issue (https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/G7ASZM6F53HVH7KXMZ4RFBLGZX47IQMB/) on the build at the time of build.

So maybe the issue will be gone with the next run and we can close this PR and the related ticket.